src/expandableCalendar/agendaList.js

```diff
@@ -170,11 +180,13 @@ class AgendaList extends Component {
         showsVerticalScrollIndicator={false}
         onViewableItemsChanged={this.onViewableItemsChanged}
         viewabilityConfig={this.viewabilityConfig}
-        renderSectionHeader={this.renderSectionHeader}
+        renderSectionHeader={this.CustomSectionHeader ? this.CustomSectionHeader : this.renderSectionHeader}
```

Comment:
Please move this check to the renderSectionHeader() method where you can safely invoke the custom render method passed in the 'renderSectionHeader' prop (see my first comment)